USERS.md

```diff
@@ -0,0 +1,5 @@
+# Project Users
+
+This document provides a WIP list of users that consume this project.
+
+- [Devfile-web](https://github.com/devfile/devfile-web)
```

Comment:
I believe this list should contain the adopters of the devfile registry whereas devfile-web contains one of the components (registry viewer) of the devfile registry. For this list, it would be good to find out what developer tools use this service, [redhat-developer/odo](https://github.com/redhat-developer/odo) would be a good example.